src/v2/Apps/Show/ShowApp.tsx

```diff
@@ -61,7 +53,7 @@ export const ShowApp: React.FC<ShowAppProps> = ({ show }) => {
             contextPageOwnerType,
           }}
         >
-          {arrivedFromFair && <ShowNavigationBanner show={show} />}
+          {isFairBooth && <ShowNavigationBanner show={show} />}
```

Comment:
**suggestion**: What do you think of renaming this component `BackToFairBanner` or something more specific than ShowNavigation since it's really only for this purpose?